src/components/Work/Tabs/Find.js

```diff
@@ -7,28 +7,28 @@ import { reactiveSearchFacets } from "../../../services/reactive-search";
 
 const styles = {
   tabContent: {
-    padding: "0 1rem"
-  }
+    padding: "0 1rem",
+  },
 };
 
 const TabsFind = ({ item }) => {
   if (!item) return;
+  const { descriptiveMetadata } = item;
   const {
-    accession_number: accessionNumber = "",
-    bibliographic_citation: bibliographicCitation = "",
-    box_name: boxName = "",
-    box_number: boxNumber = "",
-    call_number: callNumber = "",
-    catalog_key: catalogKey = "",
-    folder_name: [folderName] = "",
-    folder_number: [folderNumber] = ""
-  } = item;
+    boxName = "",
+    boxNumber = "",
+    citation: bibliographicCitation,
```

Comment:
I'd say if this `bibliographicCitation` semantic is not relevant in the newly indexed data, can we change the title?